Web/com/metacube/db/ConnectionFactory.java

```diff
@@ -0,0 +1,36 @@
+package com.metacube.db;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionFactory {
+
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/Employee";
+    private static final String USER = "root";
+    private static final String PASSWORD = "mysql";
+
+    private static Connection connection = null;
+
+    public static Connection getConnection() {
```

Comment:
This method always return new instance of Connection whenever it called. Use Singleton Design Pattern so that only first time new Connection was created and in subsequent call, previous instance of connection will return.